src/plugins/intel_gna/src/serial/gna_model_serial.cpp

```diff
@@ -63,7 +63,7 @@ inline void write_pre_processing_model(const std::shared_ptr<ov::Model>& model,
     writeString(xml_buf.str(), os);
 
     // write BIN
-    size_t ir_bin_size = bin_buf.str().size();
+    uint32_t ir_bin_size = static_cast<uint32_t>(bin_buf.str().size());
```

Comment:
please check writeBits it is template method and it uses size of template type for writing a file of model.
you cannot change a type passed there.